viewer/db.js

```diff
@@ -1719,9 +1738,10 @@ Db.getIndices = async (startTime, stopTime, bounding, rotateIndex) => {
     }
 
     // Go thru each index, convert to start/stop range and see if our time range overlaps
-    // For hourly and month indices we may search extra
+    // For hourly and month indices (and user-specified queryExtraIndices) we may search extra
     for (const iname in aliases) {
       let index = iname;
+      let isQueryExtraIndex = false;
```

Comment:
`isQueryExtraIndex` will be used to track if an index matches one of the user-defined `queryExtraIndices` patterns specified, as the logic will be slightly different. Defaults to `false`